src/deepsparse/server/server.py

```diff
@@ -140,7 +143,7 @@ def _health():
     def _add_endpoint_endpoint(cfg: EndpointConfig):
         if cfg.name is None:
             cfg.name = f"endpoint-{len(app.routes)}"
-        _add_endpoint(app, server_config, cfg, executor, context, pipeline_logger)
+        _add_endpoint(app, server_config, cfg, executor, context, server_logger)
```

Comment:
nit: could we now change these to keyword arguments? like the following: 
```
_add_endpoint(app=app, server_config=server_config, ...)
```

My train of thought being that with this, the order of calling arguments would not matter and reduce potential bugs?

I also feel it provides a first level of documentation on how to call a function.

But this is a personal preference and is not a blocker